trillian/docs/ManualDeployment.md

```diff
@@ -0,0 +1,460 @@
+[core schema file](http://github.com/google/trillian/blob/master//storage/mysql/storage.sql).
+
+
+**Cross-check**: At this point, manually connecting to the MySQL database should
+succeed:
+```
+% mysql --host=127.0.0.1 --port=3306 --user=root --database=test
+Welcome to the MariaDB monitor.  Commands end with ; or \g.
+Your MariaDB connection id is 764
+Server version: 10.1.29-MariaDB-6 Debian rodete
+
+Copyright (c) 2000, 2017, Oracle, MariaDB Corporation Ab and others.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+MariaDB [test]> show tables;
++-------------------+
+| Tables_in_test    |
++-------------------+
+| LeafData          |
+| MapHead           |
+| MapLeaf           |
+| SequencedLeafData |
+| Subtree           |
+| TreeControl       |
+| TreeHead          |
+| Trees             |
+| Unsequenced       |
++-------------------+
+9 rows in set (0.00 sec)
+
+MariaDB [test]> exit
+Bye
+```
+
+The setup so far is shown as:
+
+<img src="images/Deployment1DB.png" width="650">
+
+
+## Trillian Services
+
+The next step is to deploy two Trillian processes, the log server and the log
+signer.  These binaries are not specific to CT or to WebPKI certificates; they
+provide a general mechanism for transparently recording data in a Merkle tree.
+
+The log server (`github.com/google/trillian/server/trillian_log_server`) exposes
+a gRPC interface that allows various primitives for querying and adding to the
+underlying Merkle tree.  These operations are translated into operations on the
+storage layer, which are SQL operations in this example.
+
+ - The `--mysql_uri` option indicates where the MySQL database is available.
+ - The `--rpc_endpoint` option for the log server indicates the port that the
+   gRPC methods are available on.
+
+However, add operations are not immediately incorporated into the Merkle tree.
+Instead, pending add operations are queued up and a separate process, the log
+signer (`github.com/google/trillian/server/trillian_log_signer`) periodically
+reads pending entries from the queue.  The signer gives these entries unique,
+monotonicallly increasing, sequence numbers and incorporates them into the
+Merkle tree.
+
+ - The `--mysql_uri` option indicates where the MySQL database is available.
+ - The `--sequencer_interval`, `--batch_size` and `--num_sequencers` options
+   provide control over the timing and batching of sequencing operations.
+ - The `--force-master` option allows the signer to assume that it is the only
+   instance running (more on this [later](#primary-signer-election)).
+ - The `--logtostderr` option emits more debug logging, which is helpful while
+   getting a deployment running.
+
+<img src="images/Deployment2Trillian.png" width="650">
+
+
+## Tree Provisioning
+
+The Trillian system is *multi-tenant*: a single Trillian system can support
+multiple parallel Merkle trees.  However, this means that our particular tree
```

Comment:
parallel -> logically independent?

Reply:
went for 'multiple independent Merkle trees'